test/lib/views/eventloop-view.spec.js

```diff
@@ -1,7 +1,10 @@
 "use strict";
 
-var expect = require("chai").expect;
+var chai = require("chai");
 var sinon = require("sinon");
+var sinonChai = require("sinon-chai");
+var expect = chai.expect;
+chai.use(sinonChai);
```

Comment:
Hmm I wonder, and this could be a red herring, but is it possibly related to the order of the parameters on the npm script? I see a difference between the coverage script and test-only script.